src/pyproject_fmt/cli.py

```diff
@@ -151,6 +149,50 @@ def _build_cli() -> ArgumentParser:
     return parser
 
 
+def _settings(opt: PyProjectFmtNamespace, pyproject_toml: dict[str, Any]) -> Settings:
```

Comment:
Please move it back to inline, moving it out part of this PR makes the PR hard to review.